### ISSUE TITLE
Apparently missing `blanks/user` when closing issue

### ISSUE THREAD

**Issue by victornoel** [zerocracy/farm#1003]
- I opened an issue to ask a question to cactoos' ARC (yegor256/cactoos#841).
- The issue was resolved via discussion.
- I thus closed the issue.

`@0crat` failed at this point with the following error:
```
PID: 4@38e2c9bb-8f57-4fb1-8558-17fff4f33af4, thread: AsyncFlush-6
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/close_job.groovy
sun.reflect.GeneratedMethodAccessor1422[-1] java.lang.reflect.InvocationTargetException: null
com.zerocracy.farm.strict.StrictProject[111] java.lang.IllegalArgumentException: File "blanks/victornoel.xml" is not accessible in "PMO"
```

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
Same error here: https://github.com/yegor256/rultor/issues/1257#issuecomment-388171310

**Comment by llorllale** [comment]
I'm getting similar error when trying to remove from scope a PR [here](https://github.com/yegor256/cactoos/pull/858#issuecomment-388577256)

**Comment by yegor256** [comment]
@victornoel fixed in 5479521b6dfe985f2e33996f8ac787c323666dc1 thanks!
